package.json

```diff
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "bash ./scripts/md2html/build.sh",
-    "build-src": "npm run validate-markdown && bash ./scripts/md2html/build.sh src",
+    "build-src": "npm run validate-markdown && bash ./scripts/md2html/build.sh src && ./scripts/schema-publish.sh",
```

Comment:
> [Could you extend the build-src npm script](https://github.com/OAI/OpenAPI-Specification/pull/4376#pullrequestreview-2632281517)

I made this change, but I dislike how it makes the branch name an implicit input to the `build-src` task; eg it may, or may not build entirely to `/deploy-preview`, depending…

In `build.sh` the behavior is driven by an explicit argument.

I'm going to change the `schema-publish.sh` behavior to match, requiring `src` as an input in order to build to `/deploy-preview`, and I'll change the branch name check to a validation instead of an implicit input.